terraform/modules/hub/hub_frontend.tf

```diff
@@ -15,9 +15,45 @@ resource "aws_security_group" "frontend_task" {
 }
 
 data "template_file" "frontend_task_def" {
-  template = "${file("${path.module}/files/tasks/stub.json")}"
+  template = "${file("${path.module}/files/tasks/frontend.json")}"
 
   vars {
-    app = "frontend"
+    app           = "frontend"
+    image_and_tag = "${local.tools_account_ecr_url_prefix}-verify-frontend:latest"
+    domain        = "${var.domain}"
+  }
+}
+
+module "frontend_ecs_roles" {
+  source = "modules/ecs_iam_role_pair"
+
+  deployment       = "${var.deployment}"
+  tools_account_id = "${var.tools_account_id}"
+  service_name     = "frontend"
+  image_name       = "verify-frontend"
+}
+
+resource "aws_ecs_task_definition" "frontend" {
+  family                = "${var.deployment}-frontend"
+  container_definitions = "${data.template_file.frontend_task_def.rendered}"
+  network_mode          = "awsvpc"
```

Comment:
We're not sure what this does?

Reply:
I would read up on awsvpc networking, we'll strip it out from all of the other apps except frontend/metadata/analytics

All of the other boxes like config/policy/saml-* are one type of app per box so we can just use the box security groups

Reply:
Right, so when the docs say

> The task networking features provided by the awsvpc network mode give Amazon ECS tasks the same networking properties as Amazon EC2 instances.

They don't mean it will have the same properties as your host EC2 instance, rather that the containers networking will be treated as if itself is an EC2 instance.